packages/Core/src/panda/textStyles.ts

```diff
@@ -0,0 +1,97 @@
+import { defineTextStyles } from '@pandacss/dev'
+
+export const textStyles = defineTextStyles({
+  h0: {
+    value: {
+      fontSize: 'h0',
+      fontWeight: 'bold',
```

Comment:
...we only use the bold variant it seems.

Reply:
we use `regular` variant on `xs/sm/md/lg/subtitle-sm` keys  👀